packages/samples/react/src/components/combobox/partials/cases.tsx

```diff
@@ -1,28 +1,15 @@
-import React, { forwardRef } from 'react';
+import React from 'react';
 
 import { KolCombobox } from '@public-ui/react';
 
 import { ERROR_MSG, HINT_MSG } from '../../../shares/constants';
 import { COUNTRY_SUGGESTIONS } from '../../../shares/country';
 import type { Components } from '@public-ui/components';
 
-export const ComboboxCases = forwardRef<HTMLKolComboboxElement, Components.KolCombobox>(function InputComboboxCases(props) {
```

Comment:
Hier gab es einen Fehler in der Konsole, weil forwardRef nicht korrekt verwendet wurde. `forwardRef` schien mir überhaupt nicht notwendig, daher habe ich es ganz herausgenommen.